frontend/app/src/components/settings/api-keys/external/GnosisPayAuth.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+const { t } = useI18n();
+const understand = ref<boolean>(false);
+const name = 'gnosis_pay';
+const { loading, apiKey, actionStatus, save, confirmDelete } = useExternalApiKeys(t);
+const key = apiKey(name);
+const status = actionStatus(name);
+watchImmediate(key, (value) => {
+  if (value)
+    set(understand, true);
+});
+const premium = usePremium();
```

Comment:
```suggestion
const { t } = useI18n();

const understand = ref<boolean>(false);

const name = 'gnosis_pay';
const { loading, apiKey, actionStatus, save, confirmDelete } = useExternalApiKeys(t);
const premium = usePremium();

const key = apiKey(name);
const status = actionStatus(name);

watchImmediate(key, (value) => {
  if (value)
    set(understand, true);
});
```
It's mostly nit-picking here but it would be nice to have some space in between the sections.